Runtime/Game/LootLockerSDKManager.cs

```diff
@@ -154,6 +182,11 @@ public static string SteamSessionTicket(ref byte[] ticket, uint ticketSize)
             return sb.ToString();
         }
 
+        /// <summary>
+        /// Verify the players identity with the server and selected platform.
+        /// </summary>
+        /// <param name="deviceId"></param>
+        /// <param name="onComplete">onComplete Action</param>
```

Comment:
I'll not comment on more of these, but as a general I think it'd be better if this documented that it is a handler for the response and gets x object back.